### ISSUE TITLE
Add project.jrubyexec as a project extension

### ISSUE THREAD

**Issue by ysb33r** [jruby-gradle/jruby-gradle-plugin#23]
This will allow for jruby scripts to be executed within another task, without creating a dependency on additional tasks. This is similar to `project.exec` and `project.javaexec`. This will also slight differ in behaviour to `JRubyExec`` as task:
- The version of `jruby-complete` will be strictly tied to to `jruby.defaultVersion`. Therefore trying to set `jrubyVersion` in the `jrubyexec` closure will cause a failure
- GEMs and additional JARs will only be taken from the `jrubyExec` configuration and therefore it will not possible to supply a `configuration` parameter to the `jrubyexec` closure.
- GEMs will be installed to `jruby.gemInstallDir`. Existing gems will not be overwritten.

As with `JRubyExec`, `args`, `setArgs` and `main` are illegal within the `jrubyexec` closure. All other `javaexec` and `JRubyExec` methods should work.

_NOTE_: Once this is done, it might be the time to explore how `JRubyScriptingContainer` could be used as per ISSUE #21.
